libkbfs/keybase_service_base.go

```diff
@@ -681,3 +684,19 @@ func (k *KeybaseServiceBase) GetPublicCanonicalTLFNameAndID(
 
 	return res, nil
 }
+
+// EstablishMountDir asks the service for the current mount path
+func (k *KeybaseServiceBase) EstablishMountDir(ctx context.Context) (
+	string, error) {
+	dir, err := k.kbfsMountClient.GetCurrentMountDir(ctx)
+	if err != nil || dir == "" {
```

Comment:
The job here is "if we can't get the current mount dir, pick one from those available and save it", so I don't see why we would bail out on an error here, but I will make it do so.